src/main/java/net/snowflake/ingest/streaming/internal/FlushService.java

```diff
@@ -647,15 +645,7 @@ String getFilePath(Calendar calendar, String clientPrefix) {
             + this.counter.getAndIncrement()
             + "."
             + BLOB_EXTENSION_TYPE;
-    Path filePath =
-        Paths.get(
```

Comment:
Is there any way to pass in a separator here?

Reply:
I had the same question but unfortunately I don't think we have it